### PR TITLE
Use Code Insiders

### PR DESCRIPTION
**Pull request by orenc17**
Add a flag for using extensions and settings from VSCode Insiders

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -302,24 +302,39 @@ func rsync(src string, dest string, sshFlags string, excludePaths ...string) err
 
 func configDir() (string, error) {
 	var path string
+	var insider_path string
```
Comment (scsmithr) [inline main.go]: Should be camelCase

**Comment by foresthoffman** [comment]
@scsmithr Would it make sense to also have an `extensions-dir` and `user-data-dir` flag, since both of those are available on code-server?

https://github.com/codercom/code-server/blob/5f40ebb8457380aa1e9955302d9e41a6b042ef14/packages/server/src/cli.ts#L23-L24

**Comment by coadler** [comment]
That seems sensible but this PR is fine on its own. Should make an issue

**Comment by ammario** [comment]
Thank you for the PR, but #34 is easily implemented and will categorically solve this problem.

**Comment by coadler** [comment]
Wouldn't it make sense to have some sane defaults? These two would probably cover most use cases

**Comment by ammario** [comment]
It's unclear which path to use when both exist. I have VS Code insiders installed right now but I don't actually use it, so this change would silently break sshcode for me.

**Comment by SscSPs** [comment]
I think @ammario 's concern can be solved if we keep the selected version to be used in a config file may be.
Should be a good way of handling multiple different version

**Comment by foresthoffman** [comment]
@SscSPs I'm wary of requiring an additional file just to specify where the configuration dir is. That sounds a bit like a catch 22. Using environment variables, or at the very least CLI flags, would be much cleaner and wouldn't deviate from the standard configuration of code-server or Code.

**Comment by coadler** [comment]
We could potentially prompt users if both are found, and remember on consecutive runs

**Comment by SscSPs** [comment]
Imho doing that would be too much cli interaction when our goal is to provide a GUI text editor over SSH... 
I think that the people looking for applications like this are the ones who don't like much CLIs, otherwise they would be good with vim/Emacs/nano

**Comment by ammario** [comment]
Agreed with @foresthoffman that a configuration file is plain overkill.

A selection prompt doesn't feel very unixey to me. Will open a PR for the environment variable solution.

**Comment by orenc17** [comment]
is it better now?

**Comment by ammario** [comment]
@orenc17 how do feel about the solution in https://github.com/codercom/sshcode/pull/39 instead of this?

**Comment by orenc17** [comment]
You can integrate this PR into #39 
When the environment variable isn't set you should revert to default which will be chosen using a flag

**Comment by ammario** [comment]
Don't you think that having two ways of doing the same thing is a bad UX?

**Comment by orenc17** [comment]
IMHO the environment variables should be used only if vscode/vscode-insiders is not installed in the default path

**Comment by orenc17** [comment]
closing this to make way to #40

**Comment by orenc17** [comment]
closing this to make way for #40

**Pull request closed without merging**